app/javascript/packages/address-search/components/in-person-locations.tsx

```diff
@@ -22,20 +21,17 @@ function InPersonLocations({
   noInPersonLocationsDisplay: NoInPersonLocationsDisplay,
   resultsHeaderComponent: HeaderComponent,
 }: InPersonLocationsProps) {
-  const isPilot = locations?.some((l) => l.isPilot);
-
   if (locations?.length === 0) {
     return <NoInPersonLocationsDisplay address={address} />;
   }
 
   return (
     <>
       <h3 role="status">
-        {!isPilot &&
-          t('in_person_proofing.body.location.po_search.results_description', {
-            address,
-            count: locations?.length,
-          })}
+        {t('in_person_proofing.body.location.po_search.results_description', {
+          address,
+          count: locations?.length,
+        })}
```

Comment:
Do we need to phase this for [50/50 state](https://handbook.login.gov/articles/manage-50-50-state.html)?

Reply:
This PR doesn't include a database change, a change to anything stored in redis, a change to a job, a change to a route, or a change to a feature flag.  

Is there another 50/50 state scenario that I am missing?  Or, let me know if you do think it falls under one of the scenarios covered in 50/50 state documentation.  Since our team has hit multiple 50/50 state errors recently, I definitely agree with being cautious about it.

Reply:
Feel free to resolve this. I feel comfortable moving forward. (I didn't want to resolve it if you wanted to keep it as reminder to update docs.)

Reply:
Thanks for noting the additional 50/50 state scenario, @aduth.  Could you, or another dev who has witnessed this type of 50-50 state bug, please take a look at [this documentation PR](https://github.com/GSA-TTS/identity-handbook/pull/620)?  Thanks!

Reply:
> I think in this case it might be okay since it'll treat the absence of isPilot in the response as the same as if it had previously returned false, and the behavior would be the same?

I agree with the above.

I created a [branch](https://github.com/18F/identity-idp/compare/em/9008-test-50-50) that still uses `isPilot` in `in-person-locations.tsx`, but does not have that property present in the response from the backend.  This was meant to reproduce the 50/50 state scenario discussed in this thread.  I included a video of manually testing that [here](https://drive.google.com/file/d/1QvMwdV0g9ic6_bqumg807evCdEX2J0T5/view?usp=sharing).  I didn't hit any bugs in that manual test.  

That, along with what's discussed above, makes me think we'll be okay when this deploys.  (Let me know if anyone disagrees or thinks I missed something.  If I don't hear any objections, I'm going to move forward.)